### ISSUE TITLE
Expected outputfile [ligand.xyz] not present

### ISSUE THREAD

**Issue by ggutierrez-bio** [Acellera/htmd#69]
I got some troubles trying parameterize. It breaks at step 230 [qm_1d_scan] with this error:

`
Failed to run parameterisation : run(): Step [qm_1d_scan] did not produce the expected output: ['tor-1D-idx-*.dat', 'qm-1d-states.dat', 'n_rotamer.txt'] in directory /home/ggutierrez/.htmd/gaamp/3mpr_test/230-qm_1d_scan`

I tried a more verbose run, and beginning in the first step there are already warnings like

```
run(): Step [input] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/000-input]
completed(): Expected outputfile [ligand.xyz] not present
```

but the file is present in this folder.
I was using the latest version with this input files:
  -molecule: 3mpr_ac.mol2
  -config: pjob
  -command: parameterize --verbose --input pjob

[files.zip](https://github.com/Acellera/htmd/files/323446/files.zip)

and here, the full log:

```
$ parameterize --verbose --input pjob 
Please cite. HTMD: High-Throughput Molecular Dynamics for Molecular Discovery, J. Chem. Theory Comput., 2016, 12 (4), pp 1845-1852. 
http://pubs.acs.org/doi/abs/10.1021/acs.jctc.6b00049


You are on the latest HTMD version (1.0.27).

 === Parameterise 2016 ===

      (c) Acellera 
        AsymmetricTorsion False
                    Debug False
                   E14Fac 1.0
               Equivalent None
            ExecutionMode Inline
                 FileName 3mpr_ac.mol2
               FixCharges None
             GAUSS_SCRDIR /tmp
                  JobName 3mpr_test
                   MEMORY 8
               MaxTorsion 25
                    Model Nonpolar
             Multiplicity 1
                   NCORES 8
                NetCharge 0
                  Neutral None
                 Torsions None
       w_H_Donor_Acceptor 1.0
                  w_alpha 0.4
                 w_charge 3.0
                  w_thole 0.2
            w_water_E_min 0.4
            w_water_R_min 8.0
                 wd_alpha 0.4
                wd_charge 1.0
                 wd_thole 0.2
           wd_water_E_min 0.4
           wd_water_R_min 8.0

2016-06-20 14:43:17,982 - htmd.parameterize - INFO - Staging in input mol2/pdb file
2016-06-20 14:43:18,005 - htmd.parameterize - INFO - run(): Starting run in [/home/ggutierrez/.htmd/gaamp/3mpr_test]
2016-06-20 14:43:18,005 - htmd.parameterize - INFO - run(): Step [input] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/000-input]
2016-06-20 14:43:18,005 - htmd.parameterize - INFO - completed(): Expected outputfile [ligand.xyz] not present
2016-06-20 14:43:18,005 - htmd.parameterize - INFO - run(): Running step [input]
Running step [000] [input]
2016-06-20 14:43:18,005 - htmd.parameterize - INFO -  Running step [0]  [input] 
2016-06-20 14:43:18,148 - htmd.parameterize - INFO - run(): Step [preoptimize] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/010-preoptimize]
2016-06-20 14:43:18,148 - htmd.parameterize - INFO - completed(): Expected outputfile [mol-opt.xyz] not present
2016-06-20 14:43:18,148 - htmd.parameterize - INFO - run(): Running step [preoptimize]
Running step [010] [preoptimize]
2016-06-20 14:43:18,148 - htmd.parameterize - INFO -  Running step [10]         [preoptimize] 
Running QM Calculations: 100% (1/1) [##########################################################################################################################] eta --:-- -
2016-06-20 14:43:23,430 - htmd.parameterize - INFO - run(): Step [initial_parameters] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/020-initial_parameters]
2016-06-20 14:43:23,430 - htmd.parameterize - INFO - completed(): Expected outputfile [mol.rtf] not present
2016-06-20 14:43:23,430 - htmd.parameterize - INFO - run(): Running step [initial_parameters]
Running step [020] [initial_parameters]
2016-06-20 14:43:23,430 - htmd.parameterize - INFO -  Running step [20]         [initial_parameters] 
2016-06-20 14:43:24,898 - htmd.parameterize - INFO - run(): Step [generate_structure] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/030-generate_structure]
2016-06-20 14:43:24,898 - htmd.parameterize - INFO - completed(): Expected outputfile [mol.xpsf] not present
2016-06-20 14:43:24,898 - htmd.parameterize - INFO - run(): Running step [generate_structure]
Running step [030] [generate_structure]
2016-06-20 14:43:24,898 - htmd.parameterize - INFO -  Running step [30]         [generate_structure] 
2016-06-20 14:43:25,088 - htmd.parameterize - INFO - run(): Step [esp] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/110-esp]
2016-06-20 14:43:25,088 - htmd.parameterize - INFO - completed(): Expected outputfile [mol.xpsf] not present
2016-06-20 14:43:25,089 - htmd.parameterize - INFO - run(): Running step [esp]
Running step [110] [esp]
2016-06-20 14:43:25,089 - htmd.parameterize - INFO -  Running step [110]        [esp] 
Running QM Calculations: 100% (1/1) [##########################################################################################################################] eta --:-- |
2016-06-20 14:43:41,180 - htmd.parameterize - INFO - run(): Step [fit_esp] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/120-fit_esp]
2016-06-20 14:43:41,180 - htmd.parameterize - INFO - completed(): Expected outputfile [cg-list.txt] not present
2016-06-20 14:43:41,182 - htmd.parameterize - INFO - run(): Running step [fit_esp]
Running step [120] [fit_esp]
2016-06-20 14:43:41,182 - htmd.parameterize - INFO -  Running step [120]        [fit_esp] 
2016-06-20 14:43:41,420 - htmd.parameterize - INFO - run(): Step [fit_water] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/150-fit_water]
2016-06-20 14:43:41,420 - htmd.parameterize - INFO - completed(): Expected outputfile [soft-dih-list.txt] not present
2016-06-20 14:43:41,422 - htmd.parameterize - INFO - run(): Running step [fit_water]
Running step [150] [fit_water]
2016-06-20 14:43:41,422 - htmd.parameterize - INFO -  Running step [150]        [fit_water]
Running QM Calculations: 100% (3/3) [##########################################################################################################################] eta 00:02 |
2016-06-20 14:43:49,285 - htmd.parameterize - INFO - run(): Step [pes] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/220-pes]
2016-06-20 14:43:49,285 - htmd.parameterize - INFO - completed(): Expected outputfile [mol.prm] not present
2016-06-20 14:43:49,286 - htmd.parameterize - INFO - run(): Running step [pes]
Running step [220] [pes]
2016-06-20 14:43:49,286 - htmd.parameterize - INFO -  Running step [220]        [pes] 
2016-06-20 14:43:49,397 - htmd.parameterize - INFO - run(): Step [qm_1d_scan] in dir [/home/ggutierrez/.htmd/gaamp/3mpr_test/230-qm_1d_scan]
2016-06-20 14:43:49,397 - htmd.parameterize - INFO - completed(): Expected outputfile [mol.prm] not present
2016-06-20 14:43:49,398 - htmd.parameterize - INFO - run(): Running step [qm_1d_scan]
Running step [230] [qm_1d_scan]
2016-06-20 14:43:49,398 - htmd.parameterize - INFO -  Running step [230]        [qm_1d_scan] 
2016-06-20 14:43:49,415 - htmd.parameterize - INFO - completed(): Expected outputfile [tor-1D-idx-*.dat] not present

 Failed to run parameterisation : run(): Step [qm_1d_scan] did not produce the expected output: ['tor-1D-idx-*.dat', 'qm-1d-states.dat', 'n_rotamer.txt'] in directory /home/ggutierrez/.htmd/gaamp/3mpr_test/230-qm_1d_scan


```

**Comment by j3mdamas** [comment]
Hi Guilllermo,

I'll look into it later this afternoon.

João

**Comment by j3mdamas** [comment]
yeah, it fails because it has no actual torsions (the methyl torsion is not an torsion worth of parameterization). You can run `parameterize --torsions 3mpr_ac.mol2` and see no output.

@mj-harvey, maybe we should discuss this tomorrow, of better erroring these cases and providing the parameter files even when there are no torsions.

**Comment by stefdoerr** [comment]
It will be fixed in 1.4.0.
